README.md

```diff
@@ -300,6 +302,50 @@ ENV RES_BODY_URL_SUB                 https://<your.nexus.hostname>/repository
 > Note: By default for Nexus 3, the X-Forwarded-For headers are stripped off by the broker client so Nexus returns the npm tarball uri to the nexus registry instead of the broker server.
 Include the environment variable `REMOVE_X_FORWARDED_HEADERS=false` to disable this behavior.
 
+<br />
+
+### Nexus 2
+
+To use the Nexus 2 client with an Nexus 2 deployment (2.15 and above), run `docker pull snyk/broker:nexus2` tag. The following environment variables are needed to customize the Broker client:
```

Comment:
```suggestion
To use the Broker client with a Nexus 2 deployment (2.15 and above), run `docker pull snyk/broker:nexus2` tag. The following environment variables are needed to customize the Broker client:
```

Reply:
@kevin-s31, could you fix it for Nexus 3 (To use the Broker client with a Nexus...) as well? Thanks.